### PR TITLE
CI: Adjust + document version bump workflow

### PR DESCRIPTION
**Pull request by tooomm**
Changes:
- Little doc about how this works.
- Do not trigger on every comment written in all repo, but only on *submitted reviews*.
    - In case the PR creator themselves need to trigger the workflow, they can submit a *review comment* on their own PR.
- Changed keywords to ❌ and 🚢

<br>

TODO
- [ ] Update https://github.com/Cockatrice/Magic-Token/wiki/Token-updating-process-flow

### CODE REVIEW

File: .github/workflows/updateversion.yml [hunk 1]
```diff
@@ -14,14 +23,13 @@ jobs:
...
-      ( github.event_name == 'issue_comment' &&
-        github.event.issue.pull_request &&
-        ( github.event.comment.author_association == 'OWNER' ||
-          github.event.comment.author_association == 'MEMBER'
+      ( github.event_name == 'pull_request_review_comment' &&
+        ( github.event.review.author_association == 'OWNER' ||
+          github.event.review.author_association == 'MEMBER'
         ) &&
-        endsWith( github.event.comment.body, '@github-actions update version' )
+        contains( github.event.review.body, '@github-actions :ship:' )
```
Comment (tooomm) [inline .github/workflows/updateversion.yml:32]: @ebbit1q Any reason you used `endsWith` here over the more flexible `contains` as in the other condition?

**Review by tooomm — COMMENTED** [review]
This is a test review comment

**Comment by tooomm** [comment]
It looks like an code comment does also trigger `  pull_request_review: types: [submitted]`:

Untitled

**Comment by ebbit1q** [comment]
the workflow has to trigger for every comment, there is no other way

**Comment by tooomm** [comment]
>the workflow has to trigger for every comment, there is no other way

Can you explain a bit more, please? I thought this works differently, but am happy to understand better.

I moved the triggers inside the review process and prevented triggers on ordinary comments (like yours above, or mine just now).
At least that was the intention - preventing this empty runs/triggers on every comment created in any PR or issue:
Untitled
.

**Comment by ebbit1q** [comment]
what's the problem with that? the run gets skipped before it even does anything?

**Comment by tooomm** [comment]
Despite the other changes and open questions in this PR, what's the problem with changing/improving upon it?

If you insist on keeping it, sure, I still think some docs are helpful.

**Comment by ebbit1q** [comment]
I notice you've added a link to the wiki in your op https://github.com/Cockatrice/Magic-Token/wiki/Token-updating-process-flow
this guide doesn't seem to work anymore at all, the linked script made by @Psithief doesn't seem to exist anymore, it'd have to be rewritten

**Pull request closed without merging**